file-format-types.rdf

```diff
@@ -0,0 +1,202 @@
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="PageDescription">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#PaginatedTextDocument" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/PageDescription" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PageDescriptionGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="UnstructuredText">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:closeMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#PlainTextDocument" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/UnstructuredText" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="StructuredText">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Text" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/StructuredText" />
+  </udfrs:GenreFacetType>
+  <!-- Structured text genres -->
+  <udfrs:GenreFacetType rdf:ID="SourceCode">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="StructuredText" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#SourceCode" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Markup">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="StructuredText" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/MarkupText" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="HTML">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Markup" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#HTMLDocument" />
+    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300266021" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Presentation">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#Presentation" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Presentation" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PresentationGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Spreadsheet">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
```

Comment:
I think the intent of PageDescription's source classes is to encompass
texts produced by layout/word-processing programs. I wasn't able to
satisfactorily defend a distinction to our metadata librarian, anyway.

On Fri, Jun 5, 2015 at 12:12 PM, Esmé Cowles <notifications@github.com>
wrote:

> In file-format-types.rdf
> <https://github.com/duraspace/pcdm/pull/2#discussion_r31826593>:
>
> > +  <udfrs:GenreFacetType rdf:ID="HTML">
> > +    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
> > +    <rdfs:subClassOf rdf:resource="Markup" />
> > +    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#HTMLDocument" />
> > +    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300266021" />
> > +  </udfrs:GenreFacetType>
> > +  <udfrs:GenreFacetType rdf:ID="Presentation">
> > +    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
> > +    <rdfs:subClassOf rdf:resource="Document" />
> > +    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#Presentation" />
> > +    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Presentation" />
> > +    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#PresentationGenre" />
> > +  </udfrs:GenreFacetType>
> > +  <udfrs:GenreFacetType rdf:ID="Spreadsheet">
> > +    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
> > +    <rdfs:subClassOf rdf:resource="Document" />
>
> I don't think OfficeDocument is just a marketing strategy -- it's also a
> functional category of documents: the kind of documents that typical users
> are likely to produce. But I'm fine with deferring that for now, since I
> haven't heard anyone besides me support that.
>
> Though I do think that, separate from the OfficeDocument issue, the
> WordProcessingDocument is a real missing type:
> https://github.com/barmintor/pcdm/pull/6/files#diff-c8440ea0bf1d35562d43e1323c36f30dR157
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/duraspace/pcdm/pull/2/files#r31826593>.
>